swift/trainers/arguments.py

```diff
@@ -52,4 +52,7 @@ class TrainingArguments(SwiftArgumentsMixin, HfTrainingArguments):
 @dataclass
 class Seq2SeqTrainingArguments(SwiftArgumentsMixin,
                                HfSeq2SeqTrainingArguments):
-    pass
+
+    @property
+    def place_model_on_device(self):
+        return False if use_torchacc() else super().place_model_on_device
```

Comment:
Does torchacc only support LLM? Can it be written in SwiftArgumentsMixin

Reply:
Yes. currently torchacc only supports LLM.